network/network_integration_test.go

```diff
@@ -203,6 +204,56 @@ func TestNetworkIntegration_V2(t *testing.T) {
 		// Now assert that all nodes have received all transactions
 		waitForTransactions("node 1", node1.state, expectedDocLogSize)
 	})
+
+	t.Run("Parallel node sync", func(t *testing.T) {
+		key := nutsCrypto.NewTestKey("key")
+		expectedDocLogSize := 0
+
+		testDirectory := io.TestDirectory(t)
+		resetIntegrationTest()
+
+		node1 := startNode(t, "integration_node1", testDirectory, func(cfg *Config) {
+			cfg.ProtocolV1.AdvertHashesInterval = 24 * 60 * 60 * 1000
```

Comment:
you can (since recently) disable v1 using:

`cfg.Protocols = []int{2}` (just enables v2)